### PR TITLE
provided no-flash alternative for landing page.

### PR DESCRIPTION
**Pull request by stopfstedt**
refs #1.

while at it, i did some house-cleaning:
- removed obsolete alternative landing pages
- removed JS files needed to support flash in long dead browser versions.
- removed totally unnecessary includes of said JS files in pretty much all of pages.

### CODE REVIEW

**Comment by stopfstedt** [comment]
tested with Flash enabled and disabled on FF/Chrome and IE on Windows.

File: htdocs/index2.html [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-      <div id="homecopy_text"> Celebrating the rich heritage of UCSF, from its origins in early 
-        San Francisco to its place today at the forefront of biomedical 
-        technology, research, education, and health care. This history 
-        charts the development of the Schools of Dentistry, Medicine, 
-        Nursing, and Pharmacy, the Graduate Division, and the UCSF 
-        Medical Center.</div>
-    </div>
-    <div id="sidebar">
-      <div id="sidenav">
-      <?php include "search_include.php" ?> <br />
```
Comment (Trott) [inline htdocs/index2.html:88]: Yes! Remove PHP from this project!!!!

File: htdocs/index_noflash.html [hunk 2]
```diff
@@ -1,104 +0,0 @@
...
-technology, research, education, and health care. This history 
-charts the development of the Schools of Dentistry, Medicine, 
-Nursing, and Pharmacy, the Graduate Division, and the UCSF 
-Medical Center.</div>
-
-</div>
-
-<div id="sidebar">
-<div id="sidenav">
-<?php include "search_include.php" ?> <br />
```
Comment (Trott) [inline htdocs/index_noflash.html:82]: Yes! Remove PHP from this project!!!!

**Comment by stopfstedt** [comment]
tested fallback static image with Safari on iOS 6 device.

File: htdocs/index2.html (hunk 1 above)
Comment (stopfstedt) [inline htdocs/index2.html:88]: soon.

**Pull request merged**